### PR TITLE
Auk ecosystem

### PR DESCRIPTION
**Pull request by marshallswain**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -41,6 +40,52 @@
    * [Waterline](databases/waterline.md)
    * [LevelUP](databases/levelup.md)
    * [RethinkDB](databases/rethinkdb.md)
+* [Ecosystem & APIs](/ecosystem/readme.md)
+   * [authentication-popups](/ecosystem/authentication-popups/readme.md)
```
Comment (marshallswain) [inline SUMMARY.md]: I guess they can be single files.  If there's one that needs more than one page, we can split it into a folder.

Comment (daffl) [inline SUMMARY.md]: I'm thinking in that case it'd become its own section anyway (not under Ecosystem).

**Comment by daffl** [comment]
Just one question about the file name structure. Also, some pages will probably link somewhere else eventually (e.g. `feathers-rest` will link to `REST/Configuration`) but we can change that later. Let's get this started 😄

File: ecosystem/README.md [hunk 2]
```diff
@@ -18,46 +18,69 @@ Submit yours by creating a pull request.
 - [Feathers + Apollo](https://github.com/swarthout/feathers-apollo)
 - [Feathers + React + Mobx](https://github.com/foxhound87/rfx-stack)
 - [Feathers + React + Webpack](https://github.com/sscaff1/feathers-webpack-react)
-- [Observables with Angular2 and FeathersJS](https://berndsgn.ch/angular2-and-feathersjs/)
+- [Observables with Angular2 and FeathersJS](https://berndsgn.ch/angular2-and-feathersjs/readme.md)
```
Comment (ekryski) [inline ecosystem/README.md]: same here

File: ecosystem/README.md [hunk 3]
```diff
@@ -18,46 +18,69 @@ Submit yours by creating a pull request.
...
+- [feathers-mongoose](./feathers-mongoose/readme.md) ![Official Feathers Module](/img/tiny-feathers-logo.png) 
 - [feathers-mongoose-advanced](https://www.npmjs.com/package/feathers-mongoose-advanced) - Create a flexible Mongoose Service for FeathersJS. (service)
 - [feathers-mongoose-service](https://www.npmjs.com/package/feathers-mongoose-service) - Easily create a Mongoose Service for Featherjs. (service)
+- [feathers-nedb](./feathers-nedb/readme.md) ![Official Feathers Module](/img/tiny-feathers-logo.png) 
 - [feathers-nedb-dump](https://www.npmjs.com/package/feathers-nedb-dump) - Middleware for Feathers.js - dumps and restores NeDB database for a given service (middleware)
-- [feathers-objection](https://github.com/mcchrish/feathers-objection) - A service adapter for [Objection.js](http://vincit.github.io/objection.js/) - A minimal SQL ORM built on top of Knex.
+- [feathers-objection](https://github.com/mcchrish/feathers-objection) - A service adapter for [Objection.js](http://vincit.github.io/objection.js/readme.md) - A minimal SQL ORM built on top of Knex.
```
Comment (ekryski) [inline ecosystem/README.md]: this one as well

Comment (marshallswain) [inline ecosystem/README.md]: fixed

File: ecosystem/README.md (hunk 2 above)
Comment (marshallswain) [inline ecosystem/README.md]: fixed

File: ecosystem/README.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 ## Applications
 
-- [Feathers Chat](https://github.com/feathersjs/feathers-chat)
-- [Feathers React Native Chat](https://github.com/feathersjs/feathers-react-native-chat)
-- [Headstart](http://www.headstartapp.com/)
-- [GenerousTickets](https://generoustickets.com/)
-- [Taxfyle](https://www.taxfyle.com/)
+- [Feathers Chat](https://github.com/feathersjs/feathers-chat) ![Official Feathers Repo](/img/tiny-feathers-logo.png) 
+- [Feathers React Native Chat](https://github.com/feathersjs/feathers-react-native-chat) ![Official Feathers Repo](/img/tiny-feathers-logo.png) 
+- [Headstart](http://www.headstartapp.com/readme.md)
```
Comment (marshallswain) [inline ecosystem/README.md]: fixed

**Pull request merged**